masq/src/commands/configuration_command.rs

```diff
@@ -281,9 +358,24 @@ mod tests {
 |Start block:                      3456\n\
 |Past neighbors:                   neighbor 1\n\
 |                                  neighbor 2\n\
-"
+|Payment curves:                   \n\
+|                                  Balance decreases for sec:       11111\n\
+|                                  Balance to decrease from gwei:   1212\n\
+|                                  Payment grace before ban sec:    4578\n\
+|                                  Payment suggested after sec:     3333\n\
+|                                  Permanent debt allowed gwei:     11222\n\
+|                                  Unban when balance below gwei:   12000\n\
+|Rate pack:                        \n\
+|                                  Routing byte rate:               8\n\
+|                                  Routing service rate:            9\n\
+|                                  Exit byte rate:                  12\n\
+|                                  Exit service rate:               14\n\
+|Scan intervals:                   \n\
+|                                  Pending payable sec:             150\n\
+|                                  Payable sec:                     155\n\
+|                                  Receivable sec:                  250\n"
```

Comment:
I'm not sure people will be able to understand what `Balance decreases for sec` means from the description. it might also be good to spell out `seconds` or perhaps `secs` I've read this and the earlier documentation and I can't remember if `Balance decreases for sec` is the statement or if `Balance decreases for` in seconds is. If the latter placing the units in parentheses may help.
e.g.: 
- `Balance decreases for (secs):`
- `Balance to decrease from (gwei):`
... etc.